website/utils/components.tsx

```diff
@@ -0,0 +1,7 @@
+import AlicloudConfiguration from "../pages/docs/plugins/sources/alicloud/_configuration.md";
+import AlicloudAuthentication from "../pages/docs/plugins/sources/alicloud/_authentication.md";
+
+export const components = {
```

Comment:
We need to update this manually each time we add a plugin, but this will be temporary until we move everything to the Hub.

I tried using dynamic imports via `const component = await import(componentPath)` but it seems nextra doesn't support it directly in the config